eth/handler.go

```diff
@@ -162,6 +164,9 @@ func newHandler(config *handlerConfig) (*handler, error) {
 	broadcastCache, _ := lru.New(c_broadcastCacheSize)
 	h.broadcastCache = broadcastCache
 
+	subSyncQueue, _ := lru.New(100000)
```

Comment:
need to fix this size and make it a constant

Reply:
agreed, will change

Reply:
Made it a constant